recipes/xphyle/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xphyle" %}
+{% set version = "4.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: cdedacba0667ff864d9ad1a600f80734c942cceb70260735482bea2a16c18365
+
+build:
+  skip: true  # [not unix]
```

Comment:
I don't see any indication that this package does not support windows, although I might have missed something. 

If we can remove that `skip`, I think we can use `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
It just crashed when trying to build on Windows.
It depends on some stdlib functions in python that are only available on unix. These things happen when working with files.

Reply:
noarch python doesn't need to build on win. It only needs to build on linux